utils/src/test/scala/cromwell/pipeline/utils/TestContainersUtils.scala

```diff
@@ -27,16 +25,22 @@ object TestContainersUtils {
   private lazy val mongoConfig = ConfigFactory.load().getConfig("database.mongo")
   private def mongoPort: Int = mongoConfig.getInt("port")
 
-  class MongoContainer(dockerImageName: String) extends GenericContainer[MongoContainer](dockerImageName)
-
-  def getMongoContainer(mongoImageName: String = "mongo"): MongoContainer =
-    new MongoContainer(mongoImageName)
-      .withExposedPorts(mongoPort)
-      .withCommand("--replSet", "docker-rs")
-      .waitingFor(Wait.forLogMessage(".*Waiting for connections.*", 1))
-
-  def getConfigForMongoContainer(container: GenericContainer[_]): Config =
-    config("database.mongo.port" -> container.getMappedPort(mongoPort))
+  def getMongoContainer(mongoImageName: String = "mongo"): MongoDBContainer = {
+    val container = MongoDBContainer(DockerImageName.parse(mongoImageName))
+    container.start()
```

Comment:
Why does method `getMongoContainer` starts it? Who will stop this container once tests are completed?
This method should only return the container, starting it is up to caller